ProcessDashboard/Model/Data Access Layer/Task.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ProcessDashboard.Model.Data_Access_Layer
+{
+    class Task
+    {
+        String taskID { get; }
+        String taskName { get; }
+        Project project { get; }
+        DateTime completionDate { get; set; }
+        long estimatedTime { get; set; }
+        long actualTime { get; set; }
```

Comment:
The actual time of a task will come back from the REST API in minutes, written as a double value. At the moment, it will never have a fractional value (it will always come back as something point zero), but it wouldn't hurt to plan for fractional values in the database.  If you plan to convert to milliseconds before storing in the database, long is fine; but we should be consistent.